pyproject.toml

```diff
@@ -18,6 +18,8 @@ lxml = "^4.9.3"
 pytesseract = "^0.3.10"
 requests = "^2.32.3"
 coloredlogs = "^15.0.1"
+transformers = "^4.44.0"
+torch = "^2.4.0"
```

Comment:
I think I'm gonna move the dependencies into its own group so that it's possible to install and run Reaper without the for anyone that wants quickly contribute without downloading a gigabyte of dependencies ^^"

Reply:
Done via 583288b282d08b920fcc69047f7fd129e43b3950